### DISCUSSION TITLE
Is there a way to host the bot using an online hosting service?

### DISCUSSION THREAD

**Discussion by JarNoryb** [jagrosh/MusicBot#821] [Questions] [answered]
Hi there, I just wanna ask if there is a way to be able to keep the bot online through a web hosting service, and if there is, what are the procedures to do so?

**Comment by DJoepie** [comment, marked as answer]
Here you can find a list of common [hosting services](https://github.com/jagrosh/MusicBot/wiki/24-7-Hosting) (found in the wiki of this repo).

Basically all you need to do is get java on the virtual machine, download the jar file from the releases and run the nohub command mentioned in the [setup](https://github.com/jagrosh/MusicBot/wiki/Setup)
